### PR TITLE
Moving to yarn

### PR DESCRIPTION
**Pull request by rajzshkr**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+	yarn
+	./node_modules/.bin/lerna bootstrap
+	make build
+
 build:
 	./scripts/build.sh
 
 publish: build
-	lerna publish
+	./node_modules/.bin/lerna publish
```
Comment (Andarist) [inline Makefile]: were those changes needed?

**Comment by Andarist** [comment]
`package-lock.json` slipped into the comitted changes, should probably be removed in favor of `yarn.lock`

File: Makefile (hunk 1 above)
Comment (xtuc) [reply Makefile]: Yes, I had the .bin from node_modules inside my PATH by default, but if you don't have it the command will be unknown.

Changed in https://github.com/xtuc/charcodes/commit/c08d3c5906475f0e2e7b7d2b316c8abdef526e02

**Comment by xtuc** [comment]
Yes! I'll remove the package-lock.json, thanks @Andarist

**Pull request merged**